saleor/graphql/product/tests/queries/test_product_variants_query_with_where.py

```diff
@@ -67,3 +70,98 @@ def test_product_variant_filter_by_ids_empty_list(
     data = get_graphql_content(response)
     variants = data["data"]["productVariants"]["edges"]
     assert len(variants) == 0
+
+
+@pytest.mark.parametrize(
+    "stocks_to_create",
+    [[], [("1", 0), ("2", 1), ("4", 1)]],
+)
+def test_product_variant_where_in_stock(
+    api_client,
+    product_variant_list,
+    channel_USD,
+    stocks_to_create,
+    shipping_zones_with_warehouses,
+):
+    # given
+    warehouse = shipping_zones_with_warehouses[0].warehouses.first()
+    channel_USD.warehouses.add(warehouse)
+    channel_USD.save()
+    Stock.objects.all().delete()
```

Comment:
This line is unnecessary. Each test is executed inside a database transaction. If you have a 'Stocks' database that you don't need, don't delete it. Instead of deleting them, use or create different fixtures.